src/index.css

```diff
@@ -15,27 +15,28 @@
 
 a {
   font-weight: 500;
-  color: #646cff;
+  color: #1a3b5d;
   text-decoration: inherit;
 }
 a:hover {
-  color: #535bf2;
+  color: #28a2a2;
 }
 
 body {
   margin: 0;
-  display: flex;
-  place-items: center;
+  /* display: flex; */
+  /* place-items: center; */
```

Comment:
issue: there are some commented-out styles in index.css that we should clean up, remove the appropriate styling that is being replaced with the theme